src/installer.js

```diff
@@ -154,23 +98,7 @@ function createApplication (options, dir) {
 
   return fs.copy(options.src, applicationDir)
```

Comment:
This function is available in `-common` but because we hardcoded the path `usr/lib/` into that module, it can't be used here. Also, the `-windows` module requires an extra file to be copied (`Squirrel.exe`), so I think there's no reason to modify the `-common` package just to accommodate this function.